core/controllers/reader_test.py

```diff
@@ -401,55 +401,15 @@ class ExplorationParametersUnitTests(test_utils.GenericTestBase):
     def test_get_init_params(self):
         """Test the get_init_params() method."""
         independent_pc = param_domain.ParamChange(
-            'a', 'Copier', {'value': 'firstValue', 'parse_with_jinja': False})
-        dependent_pc = param_domain.ParamChange(
-            'b', 'Copier', {'value': '{{a}}', 'parse_with_jinja': True})
+            'a', 'Copier', {'value': 'firstValue'})
```

Comment:
I think you might need to keep parse_with_jinja, since we have that in our exploration data that is currently stored in the backend. If we remove it then we need to do a schema migration.

Have you written a one-off job (in a separate PR, probably) that can be run to output info about which usages are affected? I think we need to look at this in order to determine how to proceed safely.

Reply:
The thing is that as Aasif explains in his PR description in the backend we actually don't use `parse_with_jinja` for anything, it is only used in the frontend directly.

Reply:
But then what are we going to do with existing explorations in the datastore whose structure includes parse_with_jinja in subdictionaries? There needs to be a plan.

Reply:
The thing is that the setting is still used in the frontend, so we cannot get rid of it, on the other hand, it doesn't make sense to have a code for this in the backend if we are only using the backend to pass the values to the frontend.

Reply:
The bad thing is that this value is then kinda hidden in and not mentioned anywhere explicitly, I think that the ideal solution to this is to get rid of the generators (if it is possible) or refactor them to be explicitly mentioned, but since this is required for the Python 3 migration we don't have the time to do this rn. Also, I don't think that the refactoring of the generators is a priority compared to other stuff that we have rn.

Reply:
The idea was to keep the data as-is because the ParamChange occurs in the FE over [here](https://github.com/oppia/oppia/blob/175686dfa94c0d680f222d42e3fe864752de96b8/core/templates/pages/exploration-player-page/services/exploration-engine.service.ts#L161). `parse_with_jinja` in the structure remains the same in datastore since that is required in FE for param change, but it's removed from the generator in BE because the generator is only used in tests.

Reply:
But when I'm looking at this line in this file, you are creating a ParamChange object that doesn't have a 'parse_with_jinja' key in the dict. Isn't this skewing between the frontend and backend definitions? I would instead expect the ParamChange schema definition to be the same, but for the generator to just not do anything with the extra field.

Reply:
Probably it will work out that way, yes. You should explain clearly in the docstring what's going on so that other developers have the same understanding, too.

Reply:
Done